src/test/SimpleTestReporter.h

```diff
@@ -36,9 +36,21 @@ struct SimpleTestReporter : public ConsoleReporter
     }
 
     void
-    assertionStarting(AssertionInfo const& ai) override
+    sectionStarting(SectionInfo const& _sectionInfo) override
     {
         printDot();
+        ConsoleReporter::sectionStarting(_sectionInfo);
+    }
+
+    void
+    sectionEnded(SectionStats const& _sectionStats) override
```

Comment:
This function isn't doing anything but forwarding to the function that is being overloaded `ConsoleReporter::sectionEnded`. This is the default behavior of a virtual function that does not have an override, so why provide it explicitly?

Reply:
ah yes, that's a left over from a refactor. removing